examples/14-utils/plot_2D_data.py

```diff
@@ -36,15 +36,30 @@ def run(plotIt=True):
         ax1 = plt.subplot(121)
         ax2 = plt.subplot(122)
         # Plot Real Ex (scalar)
-        cont1, ax1 = Utils.plot2Ddata(xyz, Ex.real, dataloc=True,
-                                      ax=ax1, contourOpts={"cmap": "viridis"})
+        cont1, ax1, cont1l = Utils.plot2Ddata(
+            xyz, Ex.real, dataloc=True,
+            ax=ax1, contourOpts={"cmap": "viridis"},
+            ncontour=5, level=True,
+            levelOpts={'colors': 'k', 'linestyles': 'dashed', 'linewidths': 1}
```

Comment:
is it `linewidths` or `linewidth`?

Reply:
I confirm `linewidths`: with some of those matplotlib functions, you can give them sequence as argument too. Same for `linestyles`. You can also check at https://matplotlib.org/api/_as_gen/matplotlib.pyplot.contour.html